### DISCUSSION TITLE
Getting Error opening MSEdge

### DISCUSSION THREAD

**Discussion by BilleBarrett** [sandboxie-plus/Sandboxie#1477] [General]
Getting the following error 3 times and then MSEdge opens
The exception unknown software exception (0x0000409) occurred in the application at location 0x7489DE6B
can anyone help
Thank you
Bill

**Comment by BilleBarrett** [comment]
I forget to say this is a windows 32bit system with the latest Classic Sandboxie, it started with the previous version
Ok I just reinstalled 5.55.0 and everything work OK 
I only have one other Windows 10 32 bit client and I had to revert them to 5.55.0

**Comment by DavidXanatos** [comment]
Please try setting EnableWin32kHooks=n in your sandboxie.ini and let me know if that solves your issue on 32 bit chrome

**Comment by BilleBarrett** [comment]
I switched one client to your Plus version 32bit and it seemed to work ok

Could not find the EnableWin32kHooks=n on the Classic 32bit version

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891   ***@***.***
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
